guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -29,7 +29,7 @@ endif::[]
 +
 In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, click the {SmartProxy} that you want to use, and locate the *Registration* feature in the *Active features* list.
 +
-Optional: If the *Registration* feature is not enabled on your {SmartProxy}, enter the following command on the {SmartProxy} to enable it:
+Optional: If the *Registration* feature is not enabled on your {SmartProxy}, enter the following command on the {SmartProxy} to enable it (use a FQDN, not an IP address):
```

Comment:
```suggestion
Optional: If the *Registration* feature is not enabled on your {SmartProxy}, enter the following command on the {SmartProxy} to enable it. 
Use a FQDN, not an IP address:
```

Reply:
You've been out of docs so long @swadeley that you forget I'm strict with the parenthesis :)

Reply:
True, and thanks for review.